src/tests/ftest/io/dfuse_find_cmd.py

```diff
@@ -0,0 +1,215 @@
+        samples = self.params.get("samples", '/run/find_cmd/*')
+        self._height = self.params.get("height", '/run/find_cmd/*')
+        self._subdirs_per_node = self.params.get(
+            "subdirs_per_node", '/run/find_cmd/*')
+        self._files_per_node = self.params.get(
+            "files_per_node", '/run/find_cmd/*')
+        self._needles = self.params.get("needles", '/run/find_cmd/*')
+        challenger = self.params.get("challenger_path", '/run/find_cmd/*')
+        challenger_path = ""
+
+        self._dfuses = list()
+        self._containers = list()
+
+        self.add_pool(connect=False)
+
+        try:
+            mount_dirs = self._setup_containers(dfs_path, cont_count)
+            daos_dir_trees = self._crate_dir_trees(mount_dirs)
+            daos_stats = self._run_commands(dfs_path, samples, daos_dir_trees)
+
+            if challenger:
+                challenger_path = tempfile.mkdtemp(dir=challenger)
+                challenger_dirs = self._setup_challenger(
+                    cont_count, challenger_path)
+                challenger_dir_trees = self._crate_dir_trees(challenger_dirs)
+                challenger_stats = self._run_commands(
+                    challenger_path, samples, challenger_dir_trees)
+
+        except CommandFailure as error:
+            self.log.error("FindCmd Test Failed: %s", str(error))
+            self.fail("Test was expected to pass but it failed.\n")
+        except Exception as error:
+            self.log.error("FindCmd Test Failed: %s", str(error))
+            raise
+        finally:
+            self._teardown_dfuse(self._dfuses)
+            self.destroy_containers(self._containers)
+            self.pool.destroy()
+
+            if challenger and challenger_path:
+                shutil.rmtree(challenger_path, ignore_errors=True)
+
+        self.log.info("DAOS Stats")
+        daos_stats.print_stats()
+
+        if challenger:
+            self.log.info("Challenger Stats")
+            challenger_stats.print_stats()
+
+            self._compare_results(daos_stats, challenger_stats, "all_files")
+            self._compare_results(daos_stats, challenger_stats, "same_suffix")
+            self._compare_results(daos_stats, challenger_stats, "unique_file")
+
+    def _compare_results(self, daos_stats, challenger_stats, tag):
+        daos_max_time, _, _ = daos_stats.get_stat(tag)
+        challenger_max_time, _, _ = challenger_stats.get_stat(tag)
+
+        if daos_max_time >= challenger_max_time:
+            self.log.warning(
+                "Impossible, DAOS is slower running '{0}' tag".format(tag))
+        else:
+            self.log.info(
+                "DAOS is equal or faster running '{0}' tag".format(tag))
+
+    def _run_commands(self, test_path, samples, dir_trees):
+
+        profiler = general_utils.SimpleProfiler()
+        profiler.set_logger(self.log.info)
+
+        self.log.info("Sampling path: {0}".format(test_path))
```

Comment:
(pylint-logging-format-interpolation) Use % formatting in logging functions and pass the % parameters as arguments